src/main/java/com/vinyl/model/ProductCart.java

```diff
@@ -38,6 +38,11 @@
 	@Column(nullable=false)
 	private Double productPrice;
 
+	public ProductCart(Product product, Cart cart) {
```

Comment:
also add a no args constructor so that it satisfies bean conventions

Reply:
there is already a no args constructor under the getters and setters. should i move it near this one so it will be more visible?

Reply:
yes please

Reply:
done